src/main/java/com/cgdp/library/validation/Validator.java

```diff
@@ -0,0 +1,40 @@
+package com.cgdp.library.validation;
+
+import static com.cgdp.library.validation.IsbnValidator.isValidIsbn;
+
+import com.cgdp.library.exceptions.ValidationException;
+import java.time.LocalDate;
+import java.util.function.Supplier;
+
+public class Validator {
+
+    public static Long required(String paramName, Long value) {
+        validate(() -> value == null, "%s must not be null", paramName);
+        return value;
+    }
+
+    public static String requiredNotBlank(String paramName, String value) {
+        validate(() -> value.isBlank(), "%s must not be blank", paramName);
```

Comment:
Use method reference and also check if value is null, as name of the method contains `required`